### ISSUE TITLE
PHP 7.0 RC5 - always_populate_raw_post_data

### ISSUE THREAD

**Issue by craigcarnell** [Homebrew/homebrew-php#2478]
No matter what I do, I can't see to set always_populate_raw_post_data to -1 (Off) in /usr/local/etc/php/7.0/php.ini

**Comment by javian** [comment]
@craigcarnell Have you managed to resolve this ?

**Comment by javian** [comment]
After taking a a closer look the option has been removed in php7 so that's the reason.
```
always_populate_raw_post_data	"0"	
PHP_INI_ALL in PHP <= 4.2.3. Available since PHP 4.1.0. Removed in PHP 7.0.0.
```
from http://php.net/manual/en/ini.core.php
